src/main/java/org/gitlab4j/api/RepositoryApi.java

```diff
@@ -498,4 +499,19 @@ public CompareResults compare(String projectPath, String from, String to) throws
         Response response = get(Response.Status.OK, formData.asMap(), "projects", projectPath, "repository", "compare");
         return (response.readEntity(CompareResults.class));
     }
+    
+    /**
+     * Get a Pager of contributors from a project.
+     *
+     * GET /projects/:id/repository/contributors
+     *
+     * @param projectId the project to get the list of contributors for
+     * @param itemsPerPage the number of Project instances that will be fetched per page
+     * @return the list of contributors for the specified project ID
+     *
+     * @throws GitLabApiException if any exception occurs
+     */
+    public Pager<Contributor> getContributors(Integer projectId, int itemsPerPage) throws GitLabApiException {
```

Comment:
Should be returning `Pager<User>`  no need to create a new class for the results.